### PR TITLE
feat: Add specific selectors to all organization components #9466

### LINKED ISSUE #9466: Add specific selectors to all organization components
Previously we dropped a lot of specific selectors for the organization feature. The reason being, that most elements should be using the `display:contents` to avoid a broken flexbox flow (in the `cx-split-view`). This results in a DOM with a lot of `ng-component` selectors. These elements are hard to debug, and it blocks to add specific styling. While the later is not something Spartacus will do out of the box, there might be customers need to do this. 

We like to introduce explicit selectors again, and introduce CSS rules to all those selectors to add the `display:contents` rule to all of them.

### PR DESCRIPTION
**Pull request by Pucek9**
- add specific selecors
- add subtitle with context in change password page
- fix animation while closing create address form

Closes: #9466

### CODE REVIEW

**Review by tobi-or-not-tobi — DISMISSED** [review]
We've ended up with a CSS selector of 3348 characters... That can't be good ;) 

I've pushed an improved example: 
- Introduce host selector to all components that require the `display:content` rule (I've added the example in `BudgetDetailsComponent`, but it needs to be repeated in all components
- Drop both `cx-unit-list` and `cx-organization-list` element selectors, as would cause duplicate selectors **everywhere**

The first requires a small tslint change, which is justified in this case. 
-

File: feature-libs/organization/administration/components/unit/links/addresses/form/unit-address-form.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<cx-organization-form i18nRoot="unitAddress" [animateBack]="false">
+<cx-organization-form i18nRoot="unitAddress" [animateBack]="!(key$ | async)">
```
Comment (Platonn) [inline feature-libs/organization/administration/components/unit/links/addresses/form/unit-address-form.component.html:1]: Works great 👍

File: feature-libs/organization/administration/styles/_index.scss [hunk 2]
```diff
@@ -6,8 +6,7 @@
 @import './form';
 @import './page-template';
 
-cx-unit-list,
-cx-organization-list {
+.organization {
```
Comment (Platonn) [inline feature-libs/organization/administration/styles/_index.scss]: Should we also reflect the chages in line 27 for html RTL?

**Review by Platonn — APPROVED** [review]
Did QA through all known MyCompany views (to my knowledge), comparing against `develop` branch. Works the same.

By the way, it reduced redundant padding above unit list. And made consistent width of some cx-views, which were too wide before:
- user edit/create form - was too wide in Chrome
- unit address and unit address form - was too wide in Firefox

**Pull request merged**